===================================================================
RCS file: /cvsroot/gaim/gaim/src/gtkimhtml.c,v
retrieving revision 1.340
retrieving revision 1.341
diff -u -r1.340 -r1.341
--- gaim/gaim/src/gtkimhtml.c	2004/05/27 02:02:47	1.340
+++ gaim/gaim/src/gtkimhtml.c	2004/06/06 01:20:25	1.341
@@ -2197,7 +2197,8 @@
 							break;
 
 						id = gtk_imhtml_get_html_opt(tag, "ID=");
-
+						if (!id)
+							break;
 						gtk_imhtml_insert_image_at_iter(imhtml, atoi(id), iter);
 						break;
 					}
